camply/cli.py

```diff
@@ -260,6 +324,13 @@ def campgrounds(
     "equipment names include `Tent`, `RV`. `Trailer`, `Vehicle` and are "
     "not case-sensitive.",
 )
+equipment_id_argument = click.option(
+    "--equipment-id",
```

Comment:
@juftin What are your thoughts on the added complexity of having both `--equipment` and `--equipment-id`? 

`--equipment-id` is the easiest way forward to support Going To Camp's native equipment type support, in lieu of the `(equipment, length)` tuple that powers Rec.gov and Yellowstone. I'm not in love with having both, and equally not in love with overloading `--equipment` to support both rec.gov and goingtocamp.

Reply:
One middle ground alternative might be to break the `--equipment` API up into `--equipment` and `--equipment-length` (default 0, although I think `-1` is more fitting sentinel value).

After breaking up the `--equipment` API it could be overloaded such that `--equipment` takes an equipment name, i.e. trailer, tent, etc -- or an equipment ID, for providers that support specific equipment out of the box.

Reply:
I'm open to breaking it up. My idea with `--equipment` accepting a tuple of `equipment-name` and `equipment-length` is that it would allow you to specify for multiple types of equipment, with different lengths:

```shell
camply campsites \
    --rec-area 2991 \
    --start-date 2022-07-09 \
    --end-date 2022-07-17 \
    --nights 5 \
    --equipment RV 25 \
    --trailer 15
```

But now that I'm thinking about it - maybe that's not a common scenario and specifying a single equipment name (or `id`) and a single equipment length is more useful.

Reply:
:+1: I'll take a stab at implementing them as two params. Seems like a reasonable middle ground.